client/src/ui/track/JobCard.tsx

```diff
@@ -40,20 +58,32 @@ export function JobCard({
     }
   };
 
+  // Update interviewing round in the store
+  const handleInterviewRoundChange = (
```

Comment:
have a function that changes the metadata of the application not just the interview round

Reply:
Yes it has been fixed